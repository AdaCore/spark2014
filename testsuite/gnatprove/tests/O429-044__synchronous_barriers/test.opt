NT SKIP platform differences on windows
