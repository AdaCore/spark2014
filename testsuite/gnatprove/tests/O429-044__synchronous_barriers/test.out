p.adb:5:04: "Synchronous_Barrier" is not allowed in SPARK
p.adb:5:04: violation of pragma SPARK_Mode at test.adc:1
p.ads:3:09: high: possible data race when accessing variable "p.sb"
p.ads:3:09: high: with task "p.t1"
p.ads:3:09: high: with task "p.t2"
gnatprove: error during flow analysis and proof
